lib/journal.js

```diff
@@ -273,10 +267,23 @@
             render_day_header: render_day_header,
             render_reboot_separator: render_reboot_separator,
 
-            append: function(elt) { box.append($(elt)); },
-            prepend: function(elt) { box.prepend($(elt)); },
-            remove_last: function() { $(box[0].lastChild).remove(); },
-            remove_first: function() { $(box[0].firstChild).remove(); }
+            append: function(elt) {
+                box.appendChild(elt);
```

Comment:
TypeError: Argument 1 of Node.appendChild is not an object.

This breaks when `elt` is a string, which it often seems to be.